cli/cmd/plugin/unmanaged-cluster/main.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"os"
+
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/cmd"
+	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/command/plugin"
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/log"
+)
+
+var description = `Deploy and manage single-node, static, Tanzu clusters.`
+
+var descriptor = cliv1alpha1.PluginDescriptor{
+	Name:        "unmanaged-cluster",
+	Aliases:     []string{"um", "un", "unmanaged"},
```

Comment:
@stmcginnis @jpmcb I'd like to propose we make our aliases:

* `uc`
* `unmanaged`

Does that seem sensible to ya'll?

Reply:
Those looks good, but I would also argue for `um`. That would be my default assumption based on the name.

Reply:
I agree with Sean:
- `um`
- `uc`
- `unmanaged`

Reply:
sgtm, can someone make this change in a PR?

Reply:
On it.